### ISSUE TITLE
Edit button not visible enough

### ISSUE THREAD

**Issue by JobEijdems** [atomicdata-dev/atomic-data-browser#173]
The edit button is only visible when the user hovers over the right HTML element, which they might not think of when they don't know there is supposed to be an edit button.

**Comment by joepio** [comment]
That's true, but also a deliberate design decision. Editing by hovering is only one of the ways any property can be edited - you can always open the menu and press `edit`, or press `cmd+e`.
